seed/seed.json

```diff
@@ -1379,6 +1379,37 @@
             },
             "name": "BraveTranslateStudy"
         },
+        {
+            "experiments": [
+                {
+                    "feature_association": {
+                        "disable_feature": [
+                            "BraveEnableAutoTranslate"
+                        ]
+                    },
+                    "name": "AutoTranslateDisabled",
+                    "probability_weight": 100
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "channel": [
+                    "NIGHTLY",
+                    "BETA"
+                ],
+                "min_version": "107.1.47.31",
```

Comment:
Beta PR hasn't been merged yet.
So this version set in advance to cover the current nightly and a future beta.